src/operations/group.ts

```diff
@@ -1,101 +0,0 @@
-/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
-import { CommandOperation, CommandOperationOptions } from './command';
-import { EvalOperation } from './eval';
-import { Code, Document } from '../bson';
-import type { Callback } from '../utils';
-import type { Server } from '../sdam/server';
-import type { Collection } from '../collection';
-
-export type GroupOptions = CommandOperationOptions;
-
-/** @internal */
-export class GroupOperation extends CommandOperation<GroupOptions, Document> {
-  collectionName: string;
-  keys: any;
-  condition: Document;
-  initial: any;
-  reduceFunction: Code;
-  finalize: Code;
-
-  constructor(
-    collection: Collection,
-    keys: any,
-    condition: Document,
-    initial: Document,
-    reduce: any,
-    finalize: Code,
-    options?: GroupOptions
-  ) {
-    super(collection, options);
-    this.collectionName = collection.collectionName;
-    this.keys = keys;
-    this.condition = condition;
-    this.initial = initial;
-    this.finalize = finalize;
-    this.reduceFunction = reduce && reduce._bsontype === 'Code' ? reduce : new Code(reduce);
-  }
-
-  execute(server: Server, callback: Callback<Document>): void {
-    const cmd: Document = {
-      group: {
```

Comment:
This is small but: I think this file is the only place where we ever define `cmd.group`. If so, could we delete any leftover references to `cmd.group`? I remember seeing it referenced in at least one place (`commandSupportsReadConcern`) but there may be others.

Reply:
Good catch! I did some grepping and I think that was the last remaining reference to `cmd.group`.